model/repository.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require("mongoose");
+const repositorySchema = new mongoose.Schema({
+  name: String,
+  username: String,
+  githubId: String,
+  accessToken: String,
+  email: String
+})
+
+const model = mongoose.model('repository', repositorySchema);
+
+exports.newRepository = function (name, username, githubId, accessToken, email) {
+  return new Promise(function (resolve, reject) {
+    new model({
+      name: name,
+      username: username,
+      githubId: githubId,
```

Comment:
username can be changed so we can't use as a unique id. GitHub having uniqueId for each user. In order to sync with users, I'm using GitHub ID as GitHubId